Formula/wso2ei.rb

```diff
@@ -0,0 +1,31 @@
+class Wso2ei < Formula
+  desc "WSO2 Enterprise Integrator"
+  homepage "https://wso2.com/integration/"
+  url "https://dl.bintray.com/wso2/binary/wso2ei-6.4.0.zip"
+  sha256 "36ab3982cc517674ab1673c1947093bf8022e1f4fb971a2fe6fa1b3085a13917"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8"
+
+  def install
+    %w[
+      integrator
+      business-process
+      broker
+      analytics-dashboard
+      analytics-worker
+      micro-integrator
+      msf4j
+    ].each do |profile|
+      bin.install "bin/wso2ei-#{version}-#{profile}" => "wso2ei-#{profile}"
+    end
+
+    libexec.install Dir["*"]
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
+  end
+
+  test do
+    system "true"
```

Comment:
Just needs a test now.

Reply:
With this brew package, we want to just install the WSO2 runtime in the user's filesystem. To make the runtime functional the user has to "start" the server. We do not start the server as part of the installation and give the control of that to the user.

To test the installation we can check for an internal config file within the runtime. Hope that's sufficient to comply with Homebrew standards.

Thanks!

Reply:
The test needs to exercise and verify the functionality of the software. It runs sandboxed on CI, not on users systems.